charts/linkerd-smi/README.md.gotmpl

```diff
@@ -18,25 +18,22 @@ docs][linkerd-docs].
 ## Prerequisite: Linkerd Core Control-Plane
 
 Before installing the Linkerd SMI extension, The core control-plane has to
-be installed first by following the [Linkerd Install
-Guide](https://linkerd.io/2/tasks/install/).
+be installed first by following the [Linkerd Install Guide](https://linkerd.io/2/tasks/install/)
+or using its [Helm chart](https://github.com/linkerd/linkerd2/tree/main/charts/linkerd-control-plane).
```

Comment:
The install guide already has instructions for installing via the Helm method, so there's no need to modify this paragraph.

Reply:
Just reacted on your 

> However, can you please also leave the instructions for installing the control plane?

what did you have in mind than?

Reply:
no :smile: 
so rolling back the helm chart ref.

Reply:
Sorry, I think I got confused along the way. This looks good to me!